### ISSUE TITLE
Pagination for search results 

### ISSUE THREAD

**Issue by kalabo** [SharePoint/PnP-JS-Core#207]
The ability to have a simple pagination method with examples for the sharepoint search api wrapper with possibly an internal array value with the previous pages and a next page view link.  I can see this as being very useful.

**Comment by patrick-rodgers** [comment]
Great meeting you at Ignite - I'll try and start looking at this next week. A great idea, thanks for sharing it!

**Comment by koltyakov** [comment]
Even now it is possible in a pretty simple way, like:

```javascript
var searchResults = function(querytext, rowLimit, pageNum, callback) {
    var startRow = rowLimit * (pageNum - 1) + 1;
    $pnp.sp.search({ 
        Querytext: querytext,
        RowLimit: rowLimit,
        StartRow: startRow
    }).then(function(res) { 
        if (callback && typeof callback === "function") {
            callback(res);
        }
    });
};

searchResults("*", 5, 2, function(res) {
   console.log(res);
});
```

PnP-JS-Core proxies SearchQuery parameters that can be seen in [search.ts -> SearchQuery intenface](https://github.com/OfficeDev/PnP-JS-Core/blob/master/src/sharepoint/rest/search.ts).

**Comment by patrick-rodgers** [comment]
Wow, can't believe it took this long to get to this one. Just merged the getPage method in #434 and [updated the search wiki article](https://github.com/SharePoint/PnP-JS-Core/wiki/Working-With:-Search#paging-with-searchresultsgetpage) You can try it out in sp-pnp-js@2.0.5-beta.2
